src/NewPost.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from "react";
+import { data } from "./data";
+
+export class NewPost extends Component {
+  state = {
+    title: "",
+    author: "",
+    createdOn: "",
+    body: "",
+    tags: ""
+  };
+
+  createDateOnSubmit = () => {
+    const date = new Date();
+    let year = date.getFullYear();
+    let month = date.getMonth();
```

Comment:
let's take care of the `+ 1` here in the `month` assignment. then the `dateNow` string will be nice & clean